index.html

```diff
@@ -1,47 +1,160 @@
 ---
 layout: home
 ---
-<div class="main home-main">
-  <div class="card">
+<div class="panel">
+  <div class="panel-container">
     <div class="fixed-width">
+      <h2 class="text-center">What is Student Robotics?</h2>
+      <div class="row vertically-centered-row">
+        <div class="column m-6-12">
+          <img src="{{ '/images/advertising/team-in-arena.jpg' | prepend: site.baseurl }}" />
+        </div>
+        <div class="column m-6-12">
+          <p class="about-p">
```

Comment:
Thought: Should there be a little more (horizontal) padding between the text and image?

![image](https://user-images.githubusercontent.com/6527489/133922721-3ad97ccb-9a57-4750-a1c3-4b41382a490a.png)